internal/pkg/generator/service.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package generator generates a command given an ECS service or a workload.
+package generator
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/copilot-cli/internal/pkg/aws/ecs"
+)
+
+type ecsInformationGetter interface {
```

Comment:
what do you think of combining with https://github.com/aws/copilot-cli/pull/2192/files#diff-2f073fe539691e7b0436f7088bd2c9b85b21ee4e3f28741da3005005d19d404cR15

Reply:
It's probably not possible to merge them into one single interface because they contain functions with the same names. Although the signatures are different, Go doesn't support function overloading like that :(

Reply:
oh sorry i misread it. The one is aws/ecs and the other one is just ecs pkg.